scripts/apps/search/controllers/SearchMenuController.ts

```diff
@@ -100,17 +102,19 @@ export default function SearchMenuController(
 
     // init search providers
     if (get($rootScope.config, 'features.searchShortcut')) {
-        api.search_providers.query({max_results: 200, where: {is_closed: {$ne: true}}})
+        api.search_providers.query({max_results: 200, where: {is_closed: {$ne: true}}, with_privileges: true})
```

Comment:
Why `with_privileges` parameter needed?

Reply:
that's to only return providers which user can actually use, admins will have privileges for managing all providers but not necessarily be able to use them for fetching content

Reply:
Wouldn't it work to use the session on the back-end to check that?

Reply:
it wouldn't know what the user is doing, same user should see all in one case but not all in the other, so that param tell the backend what is the usecase atm

Reply:
> same user should see all in one case but not all in the other

Is that about viewing/managing? Wouldn't it be better then to have 2 privileges?

Reply:
there is a privilege for managing, for usage it's allowed per provider

Reply:
So what is that use case where the same user should see all in one case but not all in the other?

Reply:
when managing user should see all, that's based on privilege. when searching user should see it based on which providers are allowed for his user role.

Reply:
I see. I have done a similar thing with templates. I think it would be easier to understand if the query param was `manage=true`. `with_privileges` is seems vague to me.